presto-docs/src/main/sphinx/release/release-0.217.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.217
+=============
+
+General Changes
+---------------
+
+* Fix poor query planning performance for queries that are submitted when the cluster does not have the minimum required number of workers available.
+* Fix an issue causing a union of a table with a ``VALUES`` statement to execute on a single machine, which may result in out of memory errors.
+* Improve the logical plan by removing unnecessary remote exchanges before window functions when the data is already available locally.
+* Add grouped execution support for window functions.
+* Add support for the ``BOOLEAN`` type for ``EXPLAIN IO`` statements.
+* Add support for choosing distribution type for semi joins based on the cost model when ``join_distribution_type='AUTOMATIC'``.
+* Add support for collecting table statistics on demand with the ``ANALYZE`` statement.
```

Comment:
What is the syntax to use this?

Reply:
It is `ANALYZE qualifiedName (WITH properties)?`, but I don't think we need to expose that level of detail.

Reply:
Currently, it's only implemented in hive connector. The syntax is:
```
ANALYZE <qualifiedName>
ANALYZE <qualifiedName> WITH (partitions = <partition_list>)
    <partition_list> = ARRAY[<partition_column_values>]
    <partition_column_values> = ARRAY[VARCHAR] 

Example:
ANALYZE table1
ANALYZE table1 With (partitions = ARRAY[ARRAY['2018-02-20']])
```